tests/vmi_networking_test.go

```diff
@@ -568,6 +568,7 @@ var _ = Describe("Networking", func() {
 			dhcpVMI.Spec.Domain.Devices.Interfaces[0].DHCPOptions = &v1.DHCPOptions{
 				BootFileName:   "config",
 				TFTPServerName: "tftp.kubevirt.io",
+				NTPServers:     []string{"127.0.0.1"},
```

Comment:
Can you please add multiple ntp servers here for the test?

Reply:
done